src/client.ts

```diff
@@ -374,7 +374,7 @@ export class Client {
       data: {
         ...params,
         inputs: [inputText],
-        use_default: true,
+        use_default: !params.prompt_id,
```

Comment:
This is needed. Otherwise, API can throw in case the user has no default model. But the model itself is not needed as it is taken directly from the prompt object.

Reply:
Upcoming PR will add support for passing a custom value for `use_default`.